### PR TITLE
Split the ci-elpi_hb makefile in two.

### PR DESCRIPTION
**Pull request by ppedrot**
This would wreak havoc with the overlay files, and elpi is sensitive enough as a plugin for this to be extremely annoying.

### CODE REVIEW

**Review by SkySkimmer — APPROVED** [review]

File: Makefile.ci [hunk 1]
```diff
@@ -135,6 +137,7 @@ ci-finmap: ci-mathcomp_1
 ci-bigenough: ci-mathcomp_1
 ci-analysis: ci-elpi_hb ci-finmap ci-bigenough
 
+ci-elpi_hb: ci-elpi ci-hb
```
Comment (SkySkimmer) [inline Makefile.ci:140]:
```suggestion
ci-hb: ci-elpi
ci-elpi_hb: ci-elpi ci-hb
```

File: Makefile.ci [hunk 2]
```diff
@@ -135,6 +137,8 @@ ci-finmap: ci-mathcomp_1
 ci-bigenough: ci-mathcomp_1
 ci-analysis: ci-elpi_hb ci-finmap ci-bigenough
 
+ci-hb: ci-elpi
+ci-elpi_hb: ci-elpi ci-hb
```
Comment (SkySkimmer) [inline Makefile.ci:141]:
```suggestion
ci-elpi_hb: ci-elpi ci-hb
	true
```
Or remove elpi_hb and split the gitlab job

Or remove elpi_hb, rename the gitlab job to just ci-hb and let it implicitly build elpi

**Comment by ppedrot** [comment]
@SkySkimmer I've tweaked the makefile a bit further, introducing a CI_VIRTUAL_TARGETS variable encompassing the builds that call several shell scripts from the ci folder. I think that some other CI jobs would benefit from this trick, I'm always annoyed when e.g. writing overlays for unicoq which comes bundled with the Mtac2 job.

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**